### ISSUE TITLE
Not updating Latest performed test on Dashboard

### ISSUE THREAD

**Issue by corasaniti** [alexjustesen/speedtest-tracker#251]
Time and date of last speedtest run not updating in dashboard with latest release 0.9.0

**Comment by alexdelprete** [comment]
Working fine here:

![image](https://user-images.githubusercontent.com/7027842/208757636-ff73d5ca-345e-4dc7-8be0-d96f6deb6519.png)

![image](https://user-images.githubusercontent.com/7027842/208757736-cc992e24-58dd-4ab1-86f3-0ae654d9ec54.png)

**Comment by corasaniti** [comment]
Sure, if you go to the results page and then back to the dashboard it refreshes, but if you stay in the dashboard it doesn't work.
I use raspberry

**Comment by alexjustesen** [comment]
Issue confirmed, the date isn't polling like the stats widgets and charts. Fix incoming 👍

**Comment by alexdelprete** [comment]
> Sure, if you go to the results page and then back to the dashboard it refreshes, but if you stay in the dashboard it doesn't work. I use raspberry

Wasn't clear from the OP.

**Comment by alexjustesen** [comment]
> > Sure, if you go to the results page and then back to the dashboard it refreshes, but if you stay in the dashboard it doesn't work. I use raspberry
> 
> Wasn't clear from the OP.

I knew immediately I had missed the polling aspect of that view, benefits of being the one who codes I suppose

**Comment by FriedCheese2006** [comment]
Sorry to stir this back up, but this is still an issue if you run a speedtest, then change the timezone to something that would be in the past.

I set this up today without setting the TZ ENV variable and ran a speed test (logged as 2/17 @ 10 PM). I updated the TZ variable, restarted the container, and ran another speed test (logged as 2/17 @ 5 PM). The dashboard never updated. I just realized what was happening and deleted the first result entry.
